orttraining/orttraining/training_ops/rocm/aten_ops/aten_ops.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "orttraining/training_ops/cpu/aten_ops/aten_op.h"
+#include "core/providers/rocm/rocm_fwd.h"
+
+namespace onnxruntime {
+namespace rocm {
+
+ONNX_OPERATOR_KERNEL_EX(ATenOp, kMSDomain, 1, kRocmExecutionProvider,
+                        KernelDefBuilder().TypeConstraint("T", DataTypeImpl::AllTensorAndSequenceTensorTypes()),
+                        onnxruntime::contrib::ATenOpBase<false>);
+
+ONNX_OPERATOR_KERNEL_EX(ATenOpGrad, kMSDomain, 1, kRocmExecutionProvider,
+                        KernelDefBuilder().TypeConstraint("T", DataTypeImpl::AllTensorAndSequenceTensorTypes()),
+                        onnxruntime::contrib::ATenOpBase<true>);
+
+}  // namespace rocm
+}  // namespace onnxruntime
```

Comment:
This file may not be needed since it can be converted from CUDA implementation.
https://github.com/microsoft/onnxruntime/blob/b4e8e9b004cb3a5b5f4b9cd59212fe7704e70538/tools/ci_build/amd_hipify.py#L179

Reply:
Removed.